src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -142,28 +142,30 @@ public string VerifyRawTransaction(string rawTx)
         }
 
         [JsonRpcMethod("eth_getTransactionByBlockHashAndIndex")]
-        public JObject? GetTransactionByBlockHashAndIndex(string blockHash, ulong index)
+        public JObject? GetTransactionByBlockHashAndIndex(string blockHash, string index)
         {
+            ulong txIndex = index.HexToUlong();
```

Comment:
Are you sure index should be in hex format only?

Reply:
https://infura.io/docs/ethereum/json-rpc/eth-getTransactionCount here is the details, where it is accepting the hex string as a input